14/umbraco-cms/fundamentals/data/content-version-cleanup.md

```diff
@@ -46,6 +46,8 @@ Additionally, it is possible to keep the feature enabled but mark specific versi
 
 It is worth noting that whilst we delete rows, we do not shrink database files or rebuild indexes. For upgraded sites with a lot of history you may wish to perform these tasks. If they are not part of your regular database maintenance plan already.
 
+If set to `true`, a scheduled job will delete historic content versions every hour, while setting it to `false` will never delete any content versions regardless of settings. For more information on this, see [EnableCleanup](../../reference/configuration/contentsettings.md#enablecleanup).
```

Comment:
I've rewritten this a bit!
And what do you think about putting it further up in the article, like right above the "How it works" heading?

```suggestion
A scheduled job will delete content versions every hour when the Content Version Cleanup Policy is enabled. When not enabled, no content versions will be deleted regardless of settings. For more information on this, see [EnableCleanup](../../reference/configuration/contentsettings.md#enablecleanup).
```